<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>flow</class>
 <widget class="QDialog" name="flow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1496</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,4">
   <item>
    <widget class="QTreeWidget" name="systemTree">
     <column>
      <property name="text">
       <string>分系统</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTableWidget" name="flowTableWidget">
         <column>
          <property name="text">
           <string>名称</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>创建时间</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>最近打开时间</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>备注</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>UUID</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeWidget" name="flowTreeWidget">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <attribute name="headerStretchLastSection">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>流程</string>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="background">
         <color>
          <red>206</red>
          <green>206</green>
          <blue>206</blue>
         </color>
        </property>
       </column>
       <column>
        <property name="text">
         <string>值</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>↓前置条件</string>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="background">
         <color>
          <red>85</red>
          <green>170</green>
          <blue>255</blue>
         </color>
        </property>
       </column>
       <column>
        <property name="text">
         <string>延时</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>备注</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>测试状态</string>
        </property>
        <property name="background">
         <color>
          <red>206</red>
          <green>206</green>
          <blue>206</blue>
         </color>
        </property>
       </column>
       <item>
        <property name="text">
         <string>热待发测试</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <item>
         <property name="text">
          <string>4301_5下发前端发射控制组合加电</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="text">
          <string>1100_1热待发测试</string>
         </property>
         <property name="text">
          <string>1000</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4301_6下发地面惯组加电</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="text">
          <string>200</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </item>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
